consistency/tool/comparison/comparison_test.json

```diff
@@ -1,33 +1,238 @@
+            "kind": "WordGroup",
+            "word_frequency": {"some":2, "noise":2}
+        },
         "words": [ "Google", "apigees", "gcloud"],
-        "closest_word_groups": null
-    }, 
+        "expected": {
+            "Google":[{"id": "noise",
+                       "kind": "WordGroup",
+                       "word_frequency": {"some":2, "noise":2}}, 1], 
+            "apigees":[{"id": "noise",
+                        "kind": "WordGroup",
+                        "word_frequency": {"some":2, "noise":2}}, 1],
+            "gcloud":[{"id": "noise",
+                       "kind": "WordGroup",
+                       "word_frequency": {"some":2, "noise":2}}, 1]}
+    },
+    "none-wordgroups":{
+        "wordgroups": null,
+        "words": [ "Google", "apigees", "gcloud"],
+        
+        "expected":null  
+    },
     "none-words":{
-        "ids": [ "google", "apigee", "Gcloud"],
+        "wordgroups": [ "google", "apigee", "Gcloud"],
         "words": null,
-        "closest_word_groups": null
-    }, 
-    "unique-terms":{
-        "ids": [ "YouTube", "PlayStore", "Stadia"],
-        "words": [ "Google", "apigees", "gcloud"],
-        "closest_word_groups": {"Google":["UNIQUE_WORD", 1], 
-                                "apigees":["UNIQUE_WORD", 1],
-                                "gcloud":["UNIQUE_WORD", 1]    
-                                }
-    }, 
+        "expected": null
+    },  
     "both-null":{
-        "ids": null,
+        "wordgroups": null,
         "words": null,
-        "closest_word_groups": null
+        "expected": null
+    },
+
+    "report-test-with-unqiue":{
+        "words":["userid"],
+        "wordgroups": [
+            {
+                "id": "Userid",
+                "kind":"WordGroup",
+                "word_frequency":{
+                    "Userid": 1
+                }
+            }
+
+        ], 
+        "noisegroup":{
+                "id": "NOISE_WORDS",
+                "kind":"WordGroup",
+                "word_frequency":{
+                    "Google": 1
+                }
+            
+        },
+        "expected_report":{
+            "id": "consistency-report", 
+            "kind": "ConsistencyReport",
+            "current_variations":[
+                {
+                    "term": "userid", 
+                    "cluster":{
+                        "id": "Userid",
+                        "kind":"WordGroup",
+                        "word_frequency":{
+                            "Userid": 1
+                        }
+                    }
+         
+                }
+            ],
+            "unique_terms": ["userid"]
+        }
+    }, 
+    "report-test-with-no-unqiue":{
+        "words":["userid"],
+        "wordgroups": [
+            {
+                "id": "Userid",
+                "kind":"WordGroup",
+                "word_frequency":{
+                    "Userid": 1
+                }
+            }
+
+        ], 
+        "noisegroup":{
```

Comment:
This testcase doesn't look valid, the term "userid" won't belong to noise ever if there is already a cluster with "Userid". 

Maybe put some other word in noise and change the `words` entry 

or  

Put "userid" in the wordgroups.